install/get-teamware.sh

```diff
@@ -0,0 +1,124 @@
+#!/bin/bash
+#
+# This is a quick-start script that will download the minimal files required
+# to deploy GATE Teamware using docker compose, prompt the user for the
+# necessary configuration options and create an initial .env file
+#
+
+# GATE Teamware version that this script will install
+DEFAULT_IMAGE_TAG=latest
```

Comment:
This gets replaced by the release tag by the `create-release` workflow, so the script published as an artifact in a given release will install the version of Teamware corresponding to that specific release.